runtime/src/snapshot_utils.rs

```diff
@@ -39,54 +39,69 @@ use {
     thiserror::Error,
 };
 
+/// Trait to query the snapshot archive information
+pub trait SnapshotArchiveInfo {
```

Comment:
maybe rename to `SnapshotArchiveInfoGetter`

Reply:
Done!